cluster_pack/filesystem.py

```diff
@@ -203,6 +230,19 @@ def put(self, filename: str, path: str, chunk: int = 2**16) -> None:
                     target.write(out)
         _preserve_acls(self.base_fs, filename, path)
 
+    def put_atomic(self,
+         source_path: str,
+         destination_path: str) -> None:
+        upload_dir = f"{destination_path}.{uuid.uuid4()}.tmp"
+        try:
+            self.put(source_path, upload_dir)
+            self.mv(upload_dir, destination_path)

```

Comment:
I couldn't find the mv function in https://arrow.apache.org/docs/6.0/python/generated/pyarrow.fs.FileSystem.html, isn't it move ?

Reply:
We are using the deprecated https://arrow.apache.org/docs/6.0/python/filesystems_deprecated.html